src/figuresClasses.ts

```diff
@@ -1,19 +1,108 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: Function;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape;
 
-}
+  a: number;
+
+  b: number;
+
+  c: number;
```

Comment:
you can just add this variables with public to constructor and don't repeat yourself after